examples/decomposition/plot_online_sparse_pca.py

```diff
@@ -0,0 +1,104 @@
+"""
```

Comment:
Don't we already have this here: http://scikit-learn.org/dev/auto_examples/decomposition/plot_faces_decomposition.html ?